files/discords.json

```diff
@@ -105,16 +105,17 @@
         ]
     },
     {
-        "id": "woverflow",
-        "code": "woverflow",
+        "id": "polyfrost",
+        "code": "polyfrost",
         "fancyname": "WWWWWWWWWWWWWWWWWWW",
-        "description": "A server to discuss / get support for W-OVERFLOW mods and other things!",
+        "description": "A server to discuss / get support for Polyfrost mods and other things!",
         "icon": "wyvest.png",
         "nicknames": [
             "wyvest",
             "wyvtilities",
             "www",
-            "qalcyo"
+            "qalcyo",
+            "polyfrost"
```

Comment:
what lmao

Reply:
oh shit i forgot to change that back

do you want me to add woverflow as a nickname?